test/e2e/auth/pod_security_policy.go

```diff
@@ -169,7 +169,7 @@ func testPrivilegedPods(tester func(pod *v1.Pod)) {
 		tester(hostipc)
 	})
 
-	if common.IsAppArmorSupported() {
+	if e2esecurity.IsAppArmorSupported() {
```

Comment:
could you move the definition of `IsAppArmorSupported` to a new file within the e2e test framework (call it something like `test/e2e/framework/checks.go`?

Reply:
done, thanks